src/pkg/cli/login_test.go

```diff
@@ -62,12 +55,13 @@ func TestInteractiveLogin(t *testing.T) {
 	temp := githubAuthService
 	accessToken := "test-token"
 	fabric := "test.defang.dev"
+	// use a temp dir for the token file
+	t.Setenv("XDG_STATE_HOME", t.TempDir())
```

Comment:
Assuming that your change below works, you could do the same thing here, right?

```suggestion
	// use a temp dir for the token file
	temp := client.StateDir
	client.StateDir = filepath.Join(t.TempDir(), "defang")

	t.Cleanup(func() { client.StateDir = temp })
```

Either way let's avoid setting `XDG_STATE_HOME` since we now know that it doesn't help.

Reply:
Oh, you're right. My bad. Thanks for letting me know.